boards/common/samd21-arduino-bootloader/Makefile.dep

```diff
@@ -0,0 +1,9 @@
+# Provide stdio over USB by default
+DEFAULT_MODULE += stdio_cdc_acm
```

Comment:
Based on the discussion in #12724 maybe add a comment on this being maybe temporary solution?

Reply:
still unaddressed